src/metakb/main.py

```diff
@@ -1,15 +1,31 @@
 """Main application for FastAPI."""
+from collections.abc import AsyncGenerator
+from contextlib import asynccontextmanager
 
 from fastapi import FastAPI, Query
 from fastapi.openapi.utils import get_openapi
 
+from metakb.log_handle import configure_logs
 from metakb.query import QueryHandler
 from metakb.version import __version__
 
+
+@asynccontextmanager
+async def lifespan(app: FastAPI) -> AsyncGenerator:
+    """Configure FastAPI instance lifespan.
+
+    :param app: FastAPI app instance
+    :return: async context handler
+    """
+    configure_logs()
+    yield
```

Comment:
not convinced that this should live here if we can avoid it, but since this is our entry point for production service, I think we have to force configuration of logs here (ideally we could set it in the lambda runtime client definition